tests/suites/test_suite_platform.function

```diff
@@ -0,0 +1,38 @@
+/* BEGIN_HEADER */
+
+/* This test module exercises the platform_* module. Since, depending on the
+ * underlying operating system, the time routines are not always reliable,
+ * this suite only performs very basic sanity checks of the timing API.
+ */
+
+#include <limits.h>
+
+#if defined(MBEDTLS_HAVE_TIME)
+#include "mbedtls/platform_time.h"
+#endif
+
+/* END_HEADER */
+
+/* BEGIN_DEPENDENCIES
+ * END_DEPENDENCIES
+ */
+
+/* BEGIN_CASE depends_on:MBEDTLS_HAVE_TIME */
+void time_get_milliseconds()
+{
+    mbedtls_ms_time_t  current = mbedtls_ms_time();
+    (void) current;
+    /* This goto is added to avoid warnings from the generated code. */
+    goto exit;
```

Comment:
We can probably do a lot better than these tests if we write a `test_delay_ms()` function that would be just `Sleep(ms)` on Windows and `usleep(ms * 1000ULL)` on many other platforms (and if not available, just don't run this test).

Then we could test `mbedtls_ms_time()` by getting a value, sleeping for 10ms, then getting another value and checking (1) it's at least `orig_ms + 10` and (2) that it's not too high (would depend on CI latency, but surely `origi_ms + 5000` would be fine.

And we could do something similar for `mbedtls_time()`, using `test_delay_ms(1000)`. Obviously this will increase how long the tests take, but I think it's worth it to get some kind of use out of these tests - currently, these tests would pass functions that (1) always return 0 and (2) return a number that decreases each time it is called. Never mind whether it's actually returning any kind of "time".